app/static/openapi.yaml

```diff
@@ -91,6 +91,15 @@ paths:
           schema:
             type: string
           required: true
+        - in: query
+          name: paid
+          required: false
+          description: Whether the data is paid or not. To search for *FREE* resources, make a `GET` request to `/search?paid=false`.
+          schema:
+            type: boolean
+            enum:
```

Comment:
People should know what a boolean is. We can remove the `enum` part of this. We can also remove it from the `List Resources` portion if you don't mind while you're in there. I didn't notice it before.